lychee-lib/src/types/cache.rs

```diff
@@ -113,6 +121,7 @@ mod tests {
     }
 
     #[test]
+
```

Comment:
Does this extra space affect here affect the `#[test]` macro in anyway?

Reply:
I don't think so, but it's a formatting issue that I should fix. Thanks!